### PR TITLE
User show #25

### PR DESCRIPTION
**Pull request by CjMoore**
Adds item show functionality, and tests. Adds test to cart test for adding item to cart from show page.

### CODE REVIEW

File: spec/features/cart/user_can_add_items_to_cart_spec.rb [hunk 1]
```diff
@@ -25,4 +25,19 @@
...
+    category = Category.create!(name: "potions")
+    item = Item.create!(title: "Love Potion",
+                        description: "make him love you",
+                        price: 10.99,
+                        category: category)
+    visit item_path(item)
+    click_on "Add to Cart"
+
+    within(".notice") do
+      expect(page).to have_content("Love Potion added to cart")
```
Comment (meyerhoferc) [inline spec/features/cart/user_can_add_items_to_cart_spec.rb]: Test looks great. Glad it's simple and to the point. we don't need to over-complicate these feature test (somewhat talking to myself here)

File: spec/features/items/user_can_view_one_item_spec.rb [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+                                 description: "a rocket from the Explosive Enterprises line by Weasleys' Wizard Wheezes",
+                                 price: 10.99 )
+
+    visit item_path(item)
+
+    expect(page).to have_content('Bang Bang Boggart Banger')
+    expect(page).to have_content("a rocket from the Explosive Enterprises line by Weasleys' Wizard Wheezes")
+    expect(page).to have_content(10.99)
+    expect(page).to have_content('toys')
+    expect(page).to have_button('Add to Cart')
```
Comment (cmacaulay) [inline spec/features/items/user_can_view_one_item_spec.rb:16]: Tests look good!

**Review by cmacaulay — APPROVED** [review]
Looks good to me!

**Comment by meyerhoferc** [comment]
I approve. I like the simplicity of the test. We might need a card to add in the image of the item on the show page

**Pull request merged**